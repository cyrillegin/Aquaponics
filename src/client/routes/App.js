import React, {Component} from 'react';
import {BrowserRouter as Router, Route} from 'react-router-dom';
// import PropTypes from 'prop-types';
import HomeContainer from './../pages/Home/HomeContainer';
import AddSensorContainer from './../pages/AddSensor/AddSensorContainer';
// Nav
import NavBar from './../components/NavBar/NavBar';

export default class App extends Component {
  static propTypes = {};

  render() {
    return (
      <Router>
        <div>
          <NavBar />

          <Route exact path="/" component={HomeContainer} />

          <Route exact path="/addsensors" component={AddSensorContainer} />

        </div>
      </Router>
    );
  }
}
