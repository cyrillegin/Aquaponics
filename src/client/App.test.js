import { mount } from 'enzyme';
import React from 'react';
import App from './App';

describe('App', () => {
  it('should render a snap shot', () => {
    const wrapper = mount(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
