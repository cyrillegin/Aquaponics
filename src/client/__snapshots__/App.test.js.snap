// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render a snap shot 1`] = `ReactWrapper {}`;
